config/installbundle/base/kustomization.yaml

```diff
@@ -22,5 +22,6 @@ resources:
   - ../components/roles
   - ../components/rolebindings
   - ../components/recorder
-  - ../components/unmanageddetector
-  - ../components/webhook
\ No newline at end of file
+  - ../components/webhook
+  # Note: unmanageddetector is only used in namespaced mode
+  #- ../components/unmanageddetector
```

Comment:
I think we want to update [this code snippet](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/dev/tasks/build-release-bundle#L68) as well, otherwise the propose-build-tag won't work either.

Reply:
Never mind. I was confused (again) by the mix use of kustomize and bash on the same file.